save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -86,8 +86,11 @@ class FixPlugin : Plugin {
         when (delta) {
             is ChangeDelta -> diffGenerator
                 .generateDiffRows(delta.source.lines, delta.target.lines)
-                .joinToString("\n") { it.oldLine }
-                .let { "[ChangeDelta, position ${delta.source.position}, lines: [$it]]" }
+                .joinToString(prefix = "ChangeDelta, position ${delta.source.position}, lines:\n", separator = "\n\n") {
+                    """+${it.newLine}
+                      |-${it.oldLine}
+                      |""".trimMargin()
```

Comment:
Yes, I also have initially looked at git diff. But I'm not sure about quotes around strings, people may think that they have quotes themselves (and git diff doesn't add them too). I'll change order of new and old lines, this makes sense, thanks.

Reply:
About quotes - controversial issue. `git` have colors for this case, we doesn't, so I tried to find a way to mark them in some other manner. For user input without any quotes it's still seems for me more readable, however, in more сomplex cases it can be confusing. 
Ok, let's don't add them.